docs/src/test-components-js.md

```diff
@@ -62,7 +62,7 @@ npm i @playwright/experimental-ct-react
 ### Create a test `src/App.spec.tsx`
 
 ```js
-import { test, expect } from '@playwright/experimental-ct-react';
+import { test, expect } from '@playwright/test';
```

Comment:
I expected to see more changes in this doc mentioning including the plugin and the typings. They'll see it in the `examples/*`, but it's worth pointing out which few lines in there are the wiring.

Reply:
I'll follow up when the story settles